ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -3142,13 +3302,18 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = D6CF4EB61FC370C000CD70C5 /* adal__automationapp__ios.xcconfig */;
 			buildSettings = {
+				GCC_OPTIMIZATION_LEVEL = 0;
+				PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.adal.automationapp;
+				USER_HEADER_SEARCH_PATHS = "$(inherited) $IDCORE_PATH/src/**";
```

Comment:
Can you move these build settings to xcconfigs please?

Reply:
done